drf_sobitie/conf/asgi.py

```diff
@@ -6,7 +6,7 @@
 
 from notifications.consumers import EventNotificationConsumer
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "drf_sobitie.settings")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "conf.settings")
```

Comment:
По хорошему, тут тоже нужно указать путь от корневого модуля. Если не будет работать, оставь как сейчас
```suggestion
os.environ.setdefault("DJANGO_SETTINGS_MODULE", "drf_sobitite.conf.settings")
```